chi2/chinotype.py

```diff
@@ -693,7 +771,7 @@ def chi2_output(self, db):
         select prefix, ccd, name, {3}, frc_{3}, {0}, frc_{0}, chisq, dir
         from ranked_data {2}
         '''.format(self.chi_name, self.pcounts, limstr, self.ref, filterStr, cutoff)
-        cols, rows = do_log_sql(db, sql, self.filter)
+        cols, rows = do_log_sql(db, sql)
```

Comment:
This is not necessary. This was broken by the above "bug fix" of a non bug. Filtering could be improved though, and at the minimum the original code obviously lacks documentation.